yrs/src/doc.rs

```diff
@@ -290,4 +291,60 @@ mod test {
         txn2.apply_update(Update::decode_v1(u.as_slice()));
         assert_eq!(counter.get(), 3); // since subscription has been dropped, update was not propagated
     }
+
+    #[test]
+    fn pending_update_integration() {
```

Comment:
This is exact scenario mentioned by David in https://github.com/y-crdt/ypy/issues/30